### ISSUE TITLE
plctag.dll not found error 

### ISSUE THREAD

**Issue by BartGuy1215** [libplctag/libplctag.NET#247]
After deployment of my project using libplctag, I ran into an issue only on SOME PC's.  On some PC's I get an exception error:

Unable to load DLL "plctag.dll": The specified module could not be found. (Exception from HRESULT: 0x8007007E)

It does get extracted and appears to be in the working directory.  If I delete it, it's replaced, so it's being extracted.

I have a Win 11 PC that it works fine on.  My development laptop (win10 64) works fine, but I have another Win10 PC that it does not.  

Not sure what to even check on this one!

Project is in VB2019 using NuGet reference package LibplcTag 1.0.13 and nativeimport 1.0.17

Any help is appreciated.

**Comment by timyhac** [comment]
Try checking antivirus and set a whitelist rule to see if that helps.

**Comment by BartGuy1215** [comment]
I'm thinking that the libPLCtag code did the extraction of the dll so the .NET components must be installed correctly.  

Good assumption??

**Comment by BartGuy1215** [comment]
Figured this one out today.

I'm using VS2019 to build my project and NuGet libPLCtag package.  It appears that the C++ runtime on SOME of the PC's that I was installing to was old.   Adding a prerequisite of - Visual C++ Redistributable for Visual Studio 2015-2022 X86 and X64 to my installer (Advance Installer V19) cured it.

While I was in there I added prerequisites for the Framework version i'm using also, to ensure the framework on the final destination PC is also up to date.

Just wanted to pass this along....

John
